Makefile

```diff
@@ -6,6 +6,23 @@
 # TODO: Add in the necessary variables, etc, to make this Makefile work.
 # TODO: Add in multi-arch stuff.
 
+define yq_cmd
```

Comment:
Would it be possible to do everything with the yq & curl from go-build? I believe that's the yq we use in our other repos.

Reply:
go-build does not have yq and curl. As operator does not currently import the Makefile from go-build I had a chat with Casey and the concensus was it was better not to import it ATM

Reply:
We do still use go-build in this repo, we just don't use the common Makefile, and I don't think we should be adding a dependency on it.

Reply:
> go-build does not have yq and curl
It has both
```
$ docker run -it -e LOCAL_USER_ID=1000  calico/go-build:v0.88 sh

$ curl --version
curl 7.74.0 ...

$ yq --version
yq version 3.4.1
```